### ISSUE TITLE
non-numeric value warning with "auto" CSS height

### ISSUE THREAD

**Issue by patrikx3** [mpdf/mpdf#294]
Looks like height: auto throws an warning!  Used to work without warning!

The throw:
![image](https://cloud.githubusercontent.com/assets/4770180/21513819/60d54148-ccbd-11e6-99d1-31de9e8f4aef.png)

The code generating the warning:
![image](https://cloud.githubusercontent.com/assets/4770180/21513821/66273a3e-ccbd-11e6-80ec-2f8f45f389ae.png)

Size issue:
![image](https://cloud.githubusercontent.com/assets/4770180/21513828/74a10f04-ccbd-11e6-90a4-c0714e479ec4.png)

The tag from mPDF:
![image](https://cloud.githubusercontent.com/assets/4770180/21513844/9b16f6d0-ccbd-11e6-863a-f0dbd96866e5.png)

This is the CSS:
![image](https://cloud.githubusercontent.com/assets/4770180/21513848/a3af897e-ccbd-11e6-9ec3-629b09258982.png)

**Comment by Klap-in** [comment]
Please share the versions of the software (mpdf, webserver,etc) you are using.

**Comment by patrikx3** [comment]
docker ubuntu:latest (16.04) - php-fpm/nginx - php 7.0.8
windows 10 pro, iis, php ntd 7.1
linux mint 18.1 ,  php-fpm/nginx php 7.0.8

**Comment by patrikx3** [comment]
It is $size = 'auto', that is the problem, shows the screenshot. Also the actual file line. mpdf/mpdf.php

**Comment by bspi-stak** [comment]
We are also having that problem what we did for the mean time was ass this in line 30648
`if (is_string($size)) {
    $size = 0;
}`

**Comment by heldchen** [comment]
also reported in #300 which got closed due to no more compatibility fixes accepted for 6.1.x

**Comment by mattm1217** [comment]
This is also an issue in Tag.php

```php
// Tag.php
1785: $pdr = $currblk['padding_right'];
1786: $pdl = $currblk['padding_left'];

// $pdr and $pdl are part of a sum with some other variables further down the script.
// $currblk['padding_right'] and $currblk['padding_left'] are also summed up further down.
```
Fixed by adding in:
```php
// Tag.php
1785: if(is_string($currblk['padding_right'])){ $currblk['padding_right'] = 0; }
1786: if(is_string($currblk['padding_left'])){ $currblk['padding_left'] = 0; }
1787: $pdr = $currblk['padding_right'];
1788: $pdl = $currblk['padding_left'];
```

Environment:
Ubuntu 16.04
Apache 2.4.18
PHP 7.1.1

**Comment by ywarnier** [comment]
As far as I understand it, this should be fixed in version 7.0.0...? (if someone can confirm that, we could close this issue)

**Comment by finwe** [comment]
Closing as stale. 

Feel free to open a new issue if the problem persist in the last version.
